addons/binding/org.openhab.binding.loxone/src/main/java/org/openhab/binding/loxone/handler/LoxoneMiniserverHandler.java

```diff
@@ -364,11 +395,18 @@ private void addChannel(List<Channel> channels, String itemType, ChannelTypeUID
 
         Set<String> tags = Collections.singleton("");
 
-        if (control instanceof LxControlPushbutton || control instanceof LxControlSwitch) {
+        if (control instanceof LxControlPushbutton || control instanceof LxControlSwitch
+                || control instanceof LxControlTimedSwitch) {
             if (category != null && category.getType() == LxCategory.CategoryType.LIGHTS) {
                 tags = Collections.singleton("Lighting");
             }
             addChannel(channels, "Switch", switchTypeId, id, label, "Switch", tags);
+            // adding a deactivation delay channel for timed switch
+            if (control instanceof LxControlTimedSwitch) {
+                ChannelUID id2 = getChannelIdForControl(control, 1);
```

Comment:
why `id2` if `id` is already taken that would be a good reason have clear name explaining what id2 refers to.